prep-html-links/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <div><a href="/contact-us/contact-us.html">Click here to go to the contact page</a></div>
```

Comment:
You should leave out the first `/` in this case. Otherwise, using something like Live Server is required.